backend/src/graphql/resolvers/UserResolver.spec.ts

```diff
@@ -100,6 +100,227 @@ describe('UserResolver', () => {
           })
         })
       })
+
+      describe('include search string', () => {
```

Comment:
```suggestion
      describe('with a given `$searchString` resolver argument', () => {
```
![image](https://github.com/user-attachments/assets/6434d66f-145f-4982-bd27-dd4c39c8b774)